File: src/main/java/EKIDS/Lessons/Lesson08/Sound/Dog.java
package EKIDS.Lessons.Lesson08.Sound;

public class Dog {
    String name;
    int age;
    double weight;
    String color;
    String voice;
    String owner;

    public Dog(String name, int age, double weight, String color, String voice, String owner) {
        this.name = name;
        this.age = age;
        this.weight = weight;
        this.color = color;
        this.voice = voice;
        this.owner = owner;
    }

    public Dog() {
    }
}

class main {
    public static void main(String[] args) {
        Dog dog1 = new Dog("Боня", 3, 3., "grey", "loud", "Глеб");
        Dog dog2 = new Dog();

        switch(dog1.age) {
            case (1):
                System.out.println("собаке 1 год");
                break;
            case(2):
                System.out.println("собаке 2 года");
                break;
            case(3):
                System.out.println("собаке 3 года");
                break;
            default:
                System.out.println("Возраст неизвестен");
                break;
        }

    }

}

File: src/main/java/EKIDS/Kotlin/Lessons/Lesson05/Dog.kt
package Kotlin

class Dog {
    var name: String? = null
    var age = 0
    var weight = 0.0
    var color: String? = null
    var voice: String? = null
    var owner: String? = null

    constructor(name: String, age: Int, weight: Double, color: String, voice: String, owner: String) {
        this.name = name
        this.age = age
        this.weight = weight
        this.color = color
        this.voice = voice
        this.owner = owner
    }

    constructor() {}
}

internal class main {
    init {
        val dog1 = Dog("Боня", 3, 3.0, "grey", "loud", "Глеб")
        val dog2 = Dog()

        when (dog1.age) {
            1 -> println("собаке 1 год")
            2 -> println("собаке 2 года")
            3 -> println("собаке 3 года")
            else -> println("Возраст неизвестен")
        }
    }

}